src/main/resources/locales/en/subscription-configure-list-confirmed.json

```diff
@@ -0,0 +1,12 @@
+{
+  "title": "Subscriptions updated",
+  "header": "Subscriptions updated",
+  "subheader": "Your subscription(s) have been updated successfully",
```

Comment:
Can you double check the grammar here. The ticket says "has been" but it is "have been" here :)